perfdash/config.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+// To add new e2e test support, you need to:
+//   1) Transform e2e performance test result into *PerfData* in k8s/kubernetes/test/e2e/perftype,
+//   and print the PerfData in e2e test log.
+//   2) Add corresponding jenkins project into *JenkinsProjects*.
+//   3) Add test description and test name into *DescriptionName*.
+
+var (
+	// JenkinsProjects are the projects PerfDash supports. Downloader will download
+	// and analyze build log from the these jenkins projects.
+	JenkinsProjects = []string{"kubernetes-e2e-gce-scalability", "kubernetes-e2e-gce-serial"}
```

Comment:
It's not jenkins projects per se now. It's GCS bucket names. Actually we need a two level structure here to allow easy federated testing: top-level GCS folder URL, and set of buckets under this folder. We need to be able to represent a structure such as:
{<google_bucket_url>/kubernetes-e2e-gce-scalability, <google_bucket_url>/kubernetes-e2e-gce-serial, <redhat_bucket_url>/some_redhat_test,...}